ledger/src/slot_stats.rs

```diff
@@ -58,6 +59,14 @@ impl SlotStats {
             ("is_rooted", self.flags.contains(SlotFlags::ROOTED), bool),
             ("is_dead", self.flags.contains(SlotFlags::DEAD), bool),
         );
+
+        info!(
+            "Repair blockstore insertion metrics slot: {},
+            turbine_fec_set_index_counts: {:?},
+            repaired_indexes: {:?},
```

Comment:
This represents all repair requests inserted into blockstore, and will be of the format: 

`
Repair blockstore insertion metrics slot: 5,
{1, 2},
`

Meaning slot 5 made requests for indexes `1 and 2`